js/packages/openinference-instrumentation-langchain/package.json

```diff
@@ -27,13 +27,19 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@arizeai/openinference-semantic-conventions": "workspace:*",
+    "@arizeai/openinference-core": "workspace:*",
     "@opentelemetry/api": "^1.9.0",
     "@opentelemetry/core": "^1.25.1",
     "@opentelemetry/instrumentation": "^0.46.0"
   },
+  "peerDependencies": {
```

Comment:
@mikeldking i realized we are importing BaseTracer to extend for the tracer - this is used as an actual dep... because of that I think we need this here, does this make sense to you?

Reply:
Yeah - in that case it is a peer dep agree